Formula/srt.rb

```diff
@@ -0,0 +1,25 @@
+class Srt < Formula
+  desc "Secure Reliable Transport"
+  homepage "http://srtalliance.org"
+  url "https://github.com/Haivision/srt/archive/v1.2.0.tar.gz"
+  sha256 "ca22506cfafa18fb1f9c6dd5ceebe5e56c1b205105b2dd3736701aeea3f4145a"
+  head "https://github.com/Haivision/srt.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
+
+  def install
+    openssl = Formula["openssl"].opt_prefix
+
+    # SRT won't work appropriately with openssl which MacOS provides.
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-openssl-includedir=#{openssl}/include",
+                          "--with-openssl-libdir=#{openssl}/lib"
+    system "make", "install"
+  end
+
+  test do
+    shell_output("#{bin}/stransmit file:///dev/null file://con/ | cat -")
```

Comment:
What is that doing? Can we match its output to something expected?

Reply:
Oh, sorry. I forgot to replace with `system`. I'd like to check return with 0.